source/infrastructure/tests/__snapshots__/aws-instance-scheduler-stack.test.ts.snap

```diff
@@ -1894,71 +1885,6 @@ exports[`AwsInstanceSchedulerStack snapshot test 1`] = `
       "Type": "AWS::DynamoDB::Table",
       "UpdateReplacePolicy": "Delete",
     },
-    "instanceschedulerlambdaLambdaFunctionServiceRoleEBF44CD1": {
```

Comment:
evidently not. I didn't try to remove it explicitly so it was likely an artifact of allowing the lambda-dynamodb construct to initialize the lambda instead of providing a pre-made one